README.md

```diff
@@ -1,17 +1,21 @@
 # Open Container Specifications
 
-This project is where the [Open Container Initiative](http://www.opencontainers.org/) Specifications are written.
-This is a work in progress.
+[Open Container Initiative](http://www.opencontainers.org/) Specifications for standards on Operating System process and application containers.
```

Comment:
What are “Operating System process … containers”?  If I'm parsing that right (“for standards on ((Operating System process) and application) containers”?).